db/seeds/alerting.rb

```diff
@@ -63,3 +63,30 @@ def create_customer_with_sub(ext_id, plan:, organization:)
     {value: 10_00, recurring: true}
   ]
 })
+
+triggered_alert = UsageMonitoring::TriggeredAlert.create!(alert:, organization:, subscription:,
+  current_value: 51,
+  previous_value: 8,
+  crossed_thresholds: [
+    {code: nil, value: 10, recurring: false}, {code: :warn, value: 25, recurring: false}, {code: :alert, value: 50, recurring: false}
+  ],
```

Comment:
Do those values make sense given that it doesn't match the actual alert thresholds ?

```
  thresholds: [
    {code: "info", value: 1000_00}
  ]
```

Or did I miss something ?

Reply:
Good question! It doesn't with the current state of the alert. 

When alerts are updated, we replace (delete_all + insert) all thresholds. That why we store the threshold fully and don't "_link to the threshold_id_". We can assume this alert was set up this way 2 months ago and updated later.
The threshold make sense with the `previous_value` / `current_value` tho.